lib/api/gimbal/containers/gimbal_ring_list.h

```diff
@@ -31,10 +31,24 @@
 #define GBL_RING_LIST_CLASS(klass) (GBL_CLASS_CAST(GblRingList, klass))  //!< Casts a GblClass to a GblRingListClass
 //! @}
 
+/*! Iterates over every entry within a GblRingList, setting the \p item variable to the current entry's value.
+    Optionally takes in a \p type parameter to specify the type of the \p item variable, which defaults to void*.
+*/
+#define GblRingList_foreach(/*list, item, type*/...)                    GblRingList_foreachDefault_(__VA_ARGS__)
```

Comment:
```suggestion
#define GblRingList_foreach(/* list, item, type=void* */...)                    GblRingList_foreachDefault_(__VA_ARGS__)
```
Do that to explicitly denote the optional argument within the arg list.